package.json

```diff
@@ -12,6 +12,7 @@
     "prepack": "oclif-dev manifest && oclif-dev readme && npm shrinkwrap",
     "pretest": "npm run lint",
     "test": "nyc mocha --forbid-only \"test/**/*.test.js\"",
+    "test:mocha": "TEST_OUTPUT=1 nyc mocha 'test/**/*.test.js'",
```

Comment:
what's this for?

Reply:
this is to provide an option for when testing locally to display the cli output from the commands because the test command doesn't